CHANGELOG.md

```diff
@@ -22,11 +22,11 @@
 
 🔧 Fixes:
 
-- Pull Request Title goes here
+- Update colour for MHCLG
```

Comment:
If possible could you leave the template behind so that future people updating the CHANGELOG can use it? I can do this for you if you'd like, lemme know :)